nvflare/app_common/widgets/external_configurator.py

```diff
@@ -65,5 +64,11 @@ def _export_all_components(self, fl_ctx: FLContext) -> dict:
         engine = fl_ctx.get_engine()
         all_components = engine.get_all_components()
         components = {i: all_components.get(i) for i in self._component_ids}
-        reserved_keys = [FLMetaKey.SITE_NAME, FLMetaKey.JOB_ID]
+        reserved_keys = [
```

Comment:
Probably not going to make a diff but it's more pythonic to use a set here by changing the square brackets to curly.

Reply:
It's okay.